### PR TITLE
Implemented results tab and MCQ creation UI

### PR DESCRIPTION
**Pull request by shiquan-lim**
Completed function previously mentioned, of implementing the results tab and also restricting access to it based on whether the user is the event creator. 

Also implemented UI for creation of MCQ challenge. Challenge creation now has a switch to enable beta functionality.

### CODE REVIEW

File: src/classmentors/components/challenges/challenges.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+    // to add/delete questions and options
+
+    //options
+    var count_o_id = 1;
+
+    //questions
+    var count_q_id = 1;
+
+    $scope.questions =[{
```
Comment (dinoboff) [inline src/classmentors/components/challenges/challenges.js:131]: Why not `this.questions` and `$ctrl.questions` to access it?

Comment (shiquan-lim) [inline src/classmentors/components/challenges/challenges.js:131]: Noted. Please close this. We will send in another PR shortly
